armory/data/datasets.py

```diff
@@ -25,6 +25,7 @@
 from armory import paths
 from armory.data.librispeech import librispeech_dev_clean_split  # noqa: F401
 from armory.data.resisc45 import resisc45_split  # noqa: F401
+from armory.data.adversarial import resisc45_patch_adversarial
```

Comment:
Add `  # noqa: F401` to the end of the line to prevent flake8 errors.
Also, name it something other than the name in your function of the same name below.